context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx

```diff
@@ -208,17 +196,15 @@ export const usePanelSet = () => {
   const { isAuthenticated, isHubmapUser } = useAppContext();
 
   const {
-    entity: { dbgap_study_url, mapped_data_access_level: accessType, mapped_status: status, uuid },
+    entity: { dbgap_study_url, mapped_data_access_level: accessType, uuid },
   } = useFlaskDataContext();
 
   const hasDbGaPStudyURL = Boolean(dbgap_study_url);
   const { status: globusURLStatus, isLoading: globusURLIsLoading } = useFetchProtectedFile(uuid);
   const hasNoAccess = globusURLStatus === 403;
   const isNonConsortium = !isHubmapUser;
-  const unfinalizedStatuses = ['New', 'Error', 'QA', 'Processing', 'Invalid'];
-  const isNotFinalized = unfinalizedStatuses.includes(status);
 
-  if (accessType === 'Public') {
+  if (accessType !== 'Protected') {
```

Comment:
The access types are [protected, consortium, or public]. Has the user already been authenticated at this point? Is it ok to show this if the accessType is consortium and the user is not authenticated?

Reply:
The user has to have been authenticated to reach this point. Unlike protected datasets, which you can visit but not access files, I believe you can only visit consortium datasets if you have access. To be safe we could rely on the status of the Globus call as we do for protected access and just adapt the message.

Reply:
Correct me if I'm wrong - my understanding is that if a user is viewing a consortium dataset, they are already logged in